scripts/prepare.js

```diff
@@ -0,0 +1,36 @@
+const { spawn } = require('child_process')
+
+function runCommand(command, args) {
```

Comment:
These will always be yarn scripts, so you can simplify this by omitting `yarn`:

```suggestion
function runYarnScript(command) {
```